src/main/java/com/tms/CarFedya/entities/Admin.java

```diff
@@ -0,0 +1,54 @@
+package com.tms.CarFedya.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Admin {
```

Comment:
looks like admin and client can be saved to the one table

Reply:
fixed